test_istgt.sh

```diff
@@ -2006,6 +2006,18 @@ run_io_timeout_test()
 		# Test to verify disconnection of replica if delay from replica is more than maxiowait
 		$ISTGTCONTROL maxiowait 5
 		$IOPING  -c 3 -B -WWW /dev/$device_name > $iopinglog
+		cmd="grep -ch 'changing IO max wait time from 60 to 120' $LOGFILE"
```

Comment:
Since this test is related to verifying env. Can we push related lines of code to `run_test_env` function so that we can avoid setting `IO_MAX_WAIT_TIME` in all the cases?

Reply:
this test is about verifying env, and also make sure that 'istgtcontrol maxiowait' changes the value